BUILD.md

```diff
@@ -37,9 +37,9 @@ Transforms can take any additional number of arguments after the table.
 Lush ships with some default transforms, which are automatically injected into
 the build environment:
 
-- `viml`, head, convert parsed spec into viml
-- `lua`, head, convert parsed spec into lua
-- `vim_compatible`, tail, remove vim-incompatible values from highlight rules
+- `lush_to_viml`, head, convert parsed spec into viml
```

Comment:
I think people are moving away from viml term in favor of "vim script". https://github.com/neovim/neovim/issues/16150

Reply:
I have *always* hated "viml", feels like I am throwing up in my mouth but was never sure which to use.

:+1: